src/index.html

```diff
@@ -12,7 +15,50 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page__budy">
```

Comment:
There is a typo in the class name. It should be `page__body` instead of `page__budy`. Class names represent the meaning of the content (not the styles or tag names).